gustaf/utils/init_helper.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any
+
+
+class SplineCanNotBeLoadedHelper():
+    """
+    Class used to have better import error handling in the case the 'splinepy'
+    package is not installed. This is necessary due to that `splinepy` is not
+    a requirement of `gustaf`, but some parts of `gustaf` need it to function.
+
+    This class makes it so that `gustaf` is able to provide all non-spline
+    based functionality with out `splinepy`, but gives also comprehensive
+    errors in case these functionalities are used without the required package.
+    """
```

Comment:
```suggestion
    """
    
    self.__message = "Parts of the spline functionality in gustaf cannot be "
       "provided because the `splinepy` package was not found. Please refer "
       "to the installation instructions for more information."
```